selenium_tests/basic_test.py

```diff
@@ -28,3 +40,54 @@ def test_zero_price_purchase(self):
             "Something went wrong. You paid for this course but are not enrolled. Contact us for help."
         )
         self.assert_console_logs()
+
+    def test_learners(self):
+        """
+        Look at the learners page
+        """
+        self.login_via_admin(self.user)
+        self.get(self.live_server_url)
+
+        Role.objects.create(
+            user=self.user,
+            program=self.program,
+            role=Staff.ROLE_ID,
+        )
+
+        page_size = settings.ELASTICSEARCH_DEFAULT_PAGE_SIZE
+        with mute_signals(post_save):
+            for _ in range((page_size * 2) - 5):
```

Comment:
I want to have enough so that there are two pages worth of results, but I also want the second page to be smaller than the first so we can assert the page changed correctly. I'll add a comment